examples/src/main/java/com/radixdlt/client/examples/RadixWalletExample.java

```diff
@@ -3,12 +3,13 @@
 import com.radixdlt.client.application.RadixApplicationAPI;
 import com.radixdlt.client.application.identity.RadixIdentities;
 import com.radixdlt.client.core.atoms.AccountReference;
-import com.radixdlt.client.core.atoms.TokenRef;
+import com.radixdlt.client.core.atoms.TokenClassReference;
```

Comment:
Personally, I do like `TokenRef` a lot better or maybe TokenClassRef, a lot shorter, simpler and takes up less space

Reply:
It's shorter, but it's inconsistent with our other naming (e.g. "AccountReference").